datafusion/core/src/datasource/listing/helpers.rs

```diff
@@ -610,98 +622,27 @@ mod tests {
             parse_partitions_for_path(
                 &ListingTableUrl::parse("file:///bucket/mytable").unwrap(),
                 &Path::from("bucket/mytable/v1/file.csv"),
-                &[String::from("mypartition")]
+                vec!["mypartition"]
             )
         );
         assert_eq!(
             Some(vec!["v1", "v2"]),
             parse_partitions_for_path(
                 &ListingTableUrl::parse("file:///bucket/mytable").unwrap(),
                 &Path::from("bucket/mytable/mypartition=v1/otherpartition=v2/file.csv"),
-                &[String::from("mypartition"), String::from("otherpartition")]
+                vec!["mypartition", "otherpartition"]
             )
         );
         assert_eq!(
             Some(vec!["v1"]),
             parse_partitions_for_path(
                 &ListingTableUrl::parse("file:///bucket/mytable").unwrap(),
                 &Path::from("bucket/mytable/mypartition=v1/otherpartition=v2/file.csv"),
-                &[String::from("mypartition")]
+                vec!["mypartition"]
             )
         );
     }
 
-    #[test]
-    fn test_path_batch_roundtrip_no_partiton() {
```

Comment:
What were these tests removed?

Reply:
This is testing logic that no longer exists, we no longer encode filenames to a RecordBatch, filter it and convert it back, instead evaluating the expressions directly